var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -83,12 +84,49 @@ class Proj(AutotoolsPackage):
     )
 
     # https://proj.org/install.html#build-requirements
-    depends_on("pkgconfig@0.9.0:", type="build", when="@6:")
     depends_on("googletest", when="@6:")
     depends_on("sqlite@3.11:", when="@6:")
     depends_on("libtiff@4.0:", when="@7:+tiff")
     depends_on("curl@7.29.0:", when="@7:+curl")
+    depends_on("pkgconfig@0.9.0:", type="build", when="@6: build_system=autotools")
+    depends_on("cmake@2.6.0:", type="build", when="build_system=cmake")
+    # Builds for Windows via CMake available only
+    # after this version - builds before exist
+    # for Windows but are targeted at VS2005
+    conflicts("platform=windows", when="@:5.0.0")
```

Comment:
So to clarify, version 4 and older don't work because the package only supports Autotools, version 5.0.0 doesn't work because it's designed for VS2005, and version 5.0.1+ works?

Reply:
So the comment is a little unclear, you're right.

Reply:
Sounds good.

Reply:
Also good to know, thanks.